code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -121,7 +124,8 @@ GLOBAL_LIST_EMPTY(PDAs)
 		hidden_uplink.interact(user)
 		return
 
-	var/dat = "<html><head><title>Personal Data Assistant</title></head><body bgcolor=\"#808000\"><style>a, a:link, a:visited, a:active, a:hover { color: #000000; }img {border-style:none;}</style>"
+	var/dat = "<!DOCTYPE html><html><head><title>Personal Data Assistant</title><link href=\"https://fonts.googleapis.com/css?family=Orbitron|Share+Tech+Mono|VT323\" rel=\"stylesheet\"></head><body bgcolor=\"" + background_color + "\"><style>body{" + font_mode + "}ul,ol{list-style-type: none;}a, a:link, a:visited, a:active, a:hover { color: #000000;text-decoration:none; }img {border-style:none;}</style>"

```

Comment:
Use string embedding with [] instead of +

Reply:
dat should be a list that is .Join()'d before browse()

Reply:
That wasn't my decision, there's hundreds, maybe thousands of instances of dat being changed (Since it's the UI of the PDA). Also I haven't a scooby how to do that.

Reply:
basically

var/datlist = new list()
update all calls to dat +=, to be datlist += "string"

Then at the end of the dat code
var/dat = datlist.Join("") or similar

Reply:
you only have to do this for the PDA by the way, not every single instance of old dat usage

Reply:
Okay, but why?